verilog/analysis/checkers/struct_union_name_style_rule.cc

```diff
@@ -81,13 +83,33 @@ void StructUnionNameStyleRule::HandleSymbol(const verible::Symbol& symbol,
     }
     const auto* identifier_leaf = GetIdentifierFromTypeDeclaration(symbol);
     const auto name = ABSL_DIE_IF_NULL(identifier_leaf)->get().text();
-    if (!verible::IsLowerSnakeCaseWithDigits(name) ||
+    // get rid of the exceptions specified by user
+    std::string name_filtered;
+    if(raw_tokens.length()) {
+      name_filtered.reserve(name.length());
+      const auto &name_split = absl::StrSplit(name, '_');
+      const auto &exceptions = absl::StrSplit(raw_tokens, ',');
```

Comment:
Maybe we can already pre-process the raw_tokens in Configure() to be vector of exceptions ?

Reply:
I just reimplemented the code to your suggestions.
Could you take a look at new version?